src/components/common/ThemeProvider/index.tsx

```diff
@@ -0,0 +1,9 @@
+'use client';
```

Comment:
is this necessary in the context ?

Reply:
@akwanmaroso I dont think so, do I just delete it?

Reply:
If it's not needed, just remove it